pandas/compat/__init__.py

```diff
@@ -69,6 +75,19 @@ def str_to_bytes(s, encoding=None):
     def bytes_to_str(b, encoding=None):
         return b.decode(encoding or 'utf-8')
 
+    def strlen(data, encoding=None):
+        # encoding is for compat with PY2
+        return len(data)
+
+    def east_asian_len(data, encoding=None):
+        """
+        Calcurate display width considering unicode East Asian Width
+        """
+        if isinstance(data, text_type):
```

Comment:
It's mostly personal taste, but it might be more readable to place this function after `text_type` is defined.